### PR TITLE
[14.0][ADD] New addon web_pivot_hide_total

### PR DESCRIPTION
**Pull request by regispirard**
This addon adds a new inherited version of pivot view which hides the last "total" column and its header.

See also README file for full info.

### CODE REVIEW

File: web_pivot_hide_total/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1,13 @@
+In some cases, we want to use the columns of a pivot view
+to compare datas, but the total sum displayed in the last column
+juste makes no sens.
+
+This module intends to hide that last total column when required.
+
+.. image:: https://raw.githubusercontent.com/OCA/web/14.0/web_pivot_hide_total/static/description/before.png
```
Comment (legalsylvain) [inline web_pivot_hide_total/readme/DESCRIPTION.rst]: please replace by 

``.. image:: ../static/description/before.png``

File: web_pivot_hide_total/readme/DESCRIPTION.rst [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  :alt: Before
+
+\
+
+.. image:: https://raw.githubusercontent.com/OCA/web/14.0/web_pivot_hide_total/static/description/after.png
```
Comment (legalsylvain) [inline web_pivot_hide_total/readme/DESCRIPTION.rst]: same here.

**Comment by regispirard** [comment]
Thank you for your review @legalsylvain , I did the correction as you suggested.

**Review by sbejaoui — APPROVED** [review]
Works well, thank you

**Review by jdoutreloux — APPROVED** [review]

**Pull request merged**